R/collectors.R

```diff
@@ -138,8 +138,23 @@ col_skip <- function() {
 #' @family parsers
 #' @export
 #' @examples
-#' parse_number("$1000")
+#' ## These all return 1000
+#' parse_number("$1,000") ## leading $ and grouping character , ignored
+#' parse_number("£1000")  ## leading non-numeric £ ignored (locale doesn't matter)
+#' parse_number("AA1000") ## all leading non-numeric characters ignored
+#'
+#'
+#' ## These all return 1234567.78
+#' ## If you don't see the decimal digits, try options(digits=10)
```

Comment:
I did have to set options to see this the promised result.

So, if we invite people to change their options ... we need to show them `op <- options(digits = 10)` followed by `options(op)` (restoring the original settings). So should we just do that?

Again this is @jimhester's call.

Reply:
I am somewhat wary of setting options in examples in case the examples fail. I think just documenting this is OK in this case.